From: Anthony Sottile <asottile@umich.edu>
Date: Wed, 1 Jul 2020 10:42:28 -0700
Subject: Revert "bpo-40874 Update the required libmpdec version for the
 decimal module (GH-21202)"

This reverts commit 8bea91b5e9ea07ca93958e131b436024f0b1b1cf.
---
 Misc/NEWS.d/next/Library/2020-06-28-21-16-51.bpo-40874.YImvzA.rst | 1 -
 Modules/_decimal/_decimal.c                                       | 4 ++--
 2 files changed, 2 insertions(+), 3 deletions(-)
 delete mode 100644 Misc/NEWS.d/next/Library/2020-06-28-21-16-51.bpo-40874.YImvzA.rst

diff --git a/Misc/NEWS.d/next/Library/2020-06-28-21-16-51.bpo-40874.YImvzA.rst b/Misc/NEWS.d/next/Library/2020-06-28-21-16-51.bpo-40874.YImvzA.rst
deleted file mode 100644
index a43eab8..0000000
--- a/Misc/NEWS.d/next/Library/2020-06-28-21-16-51.bpo-40874.YImvzA.rst
+++ /dev/null
@@ -1 +0,0 @@
-The decimal module now requires libmpdec-2.5.0.
diff --git a/Modules/_decimal/_decimal.c b/Modules/_decimal/_decimal.c
index e7c44ac..0d41420 100644
--- a/Modules/_decimal/_decimal.c
+++ b/Modules/_decimal/_decimal.c
@@ -38,8 +38,8 @@
 #include "docstrings.h"
 
 
-#if !defined(MPD_VERSION_HEX) || MPD_VERSION_HEX < 0x02050000
-  #error "libmpdec version >= 2.5.0 required"
+#if !defined(MPD_VERSION_HEX) || MPD_VERSION_HEX < 0x02040100
+  #error "libmpdec version >= 2.4.1 required"
 #endif
 
 
